Missing dependency: gulp-minify-html

After a fresh installation, I tried to `gulp serve`, which did not work because of module `gulp-minify-html` missing, which is required by `partials.js`. Being deprecated anyway, I think it should be replaced with `gulp-htmlmin` if possible, or otherwise included in `package.json`.


    [14:00:35] Loading C:\dev\git\test\gulp_tasks\partials.js
    module.js:340
        throw err;
        ^
    
    Error: Cannot find module 'gulp-minify-html'
        at Function.Module._resolveFilename (module.js:338:15)
        at Function.Module._load (module.js:289:25)
        at Function.Module._load (C:\dev\git\test\node_modules\gulp-hub\lib\load-subfile.js:26:32)
        at Module.require (module.js:366:17)
        at require (module.js:385:17)
        at Object.<anonymous> (C:\dev\git\test\gulp_tasks\partials.js:2:20)
        [...]

--- a/generators/app/index.js
+++ b/generators/app/index.js
@@ -41,8 +41,9 @@ module.exports = fountain.Base.extend({
         Object.assign(pkg.devDependencies, {
           'gulp-angular-filesort': '^1.1.1',
           'gulp-angular-templatecache': '^1.8.0',
-          'gulp-ng-annotate': '^1.1.0',
-          'gulp-insert': '^0.5.0'
+          'gulp-htmlmin': '^1.3.0',
+          'gulp-insert': '^0.5.0',
+          'gulp-ng-annotate': '^1.1.0'
         });
       }
 

--- a/generators/app/templates/gulp_tasks/partials.js
+++ b/generators/app/templates/gulp_tasks/partials.js
@@ -1,5 +1,5 @@
 const gulp = require('gulp');
-const minifyHtml = require('gulp-minify-html');
+const htmlmin = require('gulp-htmlmin');
 const angularTemplatecache = require('gulp-angular-templatecache');
 <% if (modules !== 'inject') { -%>
 const insert = require('gulp-insert');
@@ -11,11 +11,7 @@ gulp.task('partials', partials);
 
 function partials() {
   return gulp.src(conf.path.src('app/**/*.html'))
-    .pipe(minifyHtml({
-      empty: true,
-      spare: true,
-      quotes: true
-    }))
+    .pipe(htmlmin())
 <% if (js === 'typescript' && modules !== 'inject') { -%>
     .pipe(angularTemplatecache('templateCacheHtml.ts', {
 <% } else { -%>

